### ISSUE TITLE
Login with Google auth and disabled cookies return CORS issue

### ISSUE THREAD

**Issue by syskin** [nuxt-community/auth-module#1310]
### Version
module: 5.0.0-1611574754.9020f2a
nuxt: 2.15

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [x] spa

### Nuxt configuration
```
  auth: {
    cookie: false,
    strategies: {
      google: {
        // ...
      },
  }
```

## Reproduction

When i define cookies option to false and i try to login through google authentication, i got CORS issue : 
``` 
localhost/:1 Access to XMLHttpRequest at 'https://www.googleapis.com/oauth2/v3/userinfo' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I tried to swith on cookies option, and it works.
I tried with facebook authentication, and with and without cookies option enable it works.

## Checklist
* [x]  I have tested with the latest Nuxt version and the issue still occurs
* [x]  I have tested with the latest module version and the issue still occurs
* [x]  I have searched the issue tracker and this issue hasn't been reported yet

### Steps to reproduce

- Configure simple auth with facebook and google
- Switch off cookies
- Try to login with both options (with correct credentials)
- Switch on cookies
- Try again with both options

### What is expected?

You should see the `userInfo` API call getting an issue with CORS when trying to login with your google account with disabled cookies option.

**Comment by bmulholland** [comment]
This sounds like an issue with Google OAuth (and even then, is probably working as intended). If you find a bug specifically with how this module handles this scenario, please open a new ticket.
